src/main.rs

```diff
@@ -177,7 +178,12 @@ fn main() {
     } else if flags.is_present("dir") {
         // Safe to unwrap because 'dir' requires 'dir_target'
         let matches = match flags.value_of("dir_target") {
-            Some(to) => frecency.directory_matches(to),
+            Some(to) => {
+                env::current_dir().map( |cwd| {
```

Comment:
That space before `|` looks odd; did you run `cargo fmt` with a recent `rustfmt-nightly` installed?

Reply:
I put it there meself, so I shall now remove it.